src/components/RegisterUser.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const RegisterUser = ( { user, setUser } ) => {
+    const [username, setUsername] = useState();
+    const [fullName, setFullName] = useState();
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+
+    const submitHandler = (e) => {
+        e.preventDefault();
+        console.log("register user - ", username, fullName, email, password);
+        console.log("ToDo: RegisterUser -> createUser( {user}, setUser )")
+    };
+
+    return(
+        <DivRegister>
+            <form onSubmit={submitHandler}>
+                {!user && <input defaultValue={"Choose a username"} onChange={(e) => setUsername(e.target.value)} />}
```

Comment:
Same as with DeleteUser.js, this component should only be available/called when the user is not logged in, as it's called from the "Register/SignUp" button and that will only be available when the user is not logged in, therefore you don't have to bother with conditionally rendering the inputs (line 19-23)

Reply:
Done mate